### ISSUE TITLE
Health checking

### ISSUE THREAD

**Issue by erikvg01** [50ButtonsEach/fliclib-linux-hci#32]
I've been attempting to sort out a way to check the 'health' of the daemon while it is running.  On rare occasion, the daemon seems to forget the contents of the database, causing the buttons to fail to connect.  When I grep out the process, it appears to be running normally.  The closest I can get is to launch the simpleclient and use the getInfo command to see if the buttons are listed.  I'd love to be able to have this same interaction from the node-red context so I can restart services etc.  Time and skill are preventing me from fully writing my own...Mostly wondering if anyone out there has built something that fits this need.

Any guidance is appreciated.  Thank you!

**Comment by Emill** [comment]
Can you tell a bit more about that the db forgets the buttons? That really
shouldn't happen.

**Comment by erikvg01** [comment]
Is there a log location that is written to by default?

Twice in the past two months, I have observed that flic's associated with a one specific Raspberry Pi (model B) start blinking red.  I connect the simpleclient and getInfo, which responds with nothing at all.

**Comment by Emill** [comment]
No, you have to manually assign a log file location, otherwise it's
discarded when running in daemon mode.

Can you clearify "blinking red"? Is it blinking or fading in and out or
some other pattern? And for how long time is it lit up?

Den 22 aug. 2016 10:10 em skrev "erikvg01" <notifications@github.com>:

**Comment by erikvg01** [comment]
I'll re-launch with a log location.  Buttons blink red once on press, then fade out.

**Comment by Emill** [comment]
Well the fading is standard and means that it tries to establish a
connection. It stops once it gets connected.

Den 22 aug. 2016 10:18 em skrev "erikvg01" <notifications@github.com>:

**Comment by erikvg01** [comment]
All presses are queued when it's in this state.  Once I restart the flicd and restart my node-red client, the queued messages dump out.
